### PR TITLE
fix: Don't break those still using ALPN multiplex (Fixes #7794)

### LINKED ISSUE #7794: jitsi-meet-turnserver 1.0.4184-1 misses `/usr/share/jitsi-meet-turnserver/jitsi-meet.conf`
Upgrading `jitsi-meet-turnserver` from 1.0.4420-1 to 1.0.4434-1 the file `/usr/share/jitsi-meet-turnserver/jitsi-meet.conf` is missing. This causes Nginx fail to start, as `/etc/nginx/modules-enabled/60-jitsi-meet.conf` is a symbolic link to that file.

### PR DESCRIPTION
**Pull request by damencho**

### CODE REVIEW

File: doc/debian/jitsi-meet/jitsi-meet.conf [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# this is jitsi-meet nginx module configuration
+# this forward all http traffic to the nginx virtual host port
+# and the rest to the turn server
+#
+# Multiplexing based on ALPN is DEPRECATED. ALPN does not play well with websockets on some browsers and reverse proxies.
+# To migrate away from using it read: https://jitsi.org/multiplexing-to-bridge-ws-howto
+# This file will be removed at some point and if deployment is still using it, will brake.
```
Comment (saghul) [inline doc/debian/jitsi-meet/jitsi-meet.conf:7]:
```suggestion
# This file will be removed at some point and if deployment is still using it, will break.
```

File: doc/debian/jitsi-meet/jitsi-meet.conf [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# this is jitsi-meet nginx module configuration
+# this forward all http traffic to the nginx virtual host port
+# and the rest to the turn server
+#
+# Multiplexing based on ALPN is DEPRECATED. ALPN does not play well with websockets on some browsers and reverse proxies.
+# To migrate away from using it read: https://jitsi.org/multiplexing-to-bridge-ws-howto
```
Comment (saghul) [inline doc/debian/jitsi-meet/jitsi-meet.conf:6]: Why not link directly to the handbook?

**Review by saghul — APPROVED** [review]
LGTM with a grammar comment.

File: doc/debian/jitsi-meet/jitsi-meet.conf (hunk 2 above)
Comment (damencho) [reply doc/debian/jitsi-meet/jitsi-meet.conf:6]: Changing the title for example will invalidate the anchor link. And I already used this link in the warning message when installing.

Comment (saghul) [reply doc/debian/jitsi-meet/jitsi-meet.conf:6]: Ack

**Review by saghul — APPROVED** [review]

**Pull request merged**